system/config/Config.cfc

```diff
@@ -222,6 +224,7 @@ component output=false {
 		settings.adminRoles.contentadmin       = [ "sites.*", "presideobject.site.*", "presideobject.link.*", "sitetree.*", "presideobject.page.*", "datamanager.*", "assetmanager.*", "presideobject.asset.*", "presideobject.asset_folder.*", "formbuilder.*", "!formbuilder.lockForm", "!formbuilder.activateForm" ];
 		settings.adminRoles.contenteditor      = [ "presideobject.link.*", "sites.navigate", "sitetree.*", "presideobject.page.*", "datamanager.*", "assetmanager.*", "presideobject.asset.*", "presideobject.asset_folder.*", "!*.delete", "!*.manageContextPerms", "!assetmanager.folders.add" ];
 		settings.adminRoles.formbuildermanager = [ "formbuilder.*" ];
+		settings.adminRoles.auditTrailmanager  = [ "auditTrail.*" ];
```

Comment:
Its good that you've made this work. However, I don't think that we need / want a specific role just for the audit trail. Instead, add "auditTrail.*" to the "sysadmin" admin role.